lnclient/ldk/ldk.go

```diff
@@ -1195,19 +1195,22 @@ func (ls *LDKService) GetOnchainBalance(ctx context.Context) (*lnclient.OnchainB
 	}, nil
 }
 
-func (ls *LDKService) RedeemOnchainFunds(ctx context.Context, toAddress string, amount uint64, sendAll bool) (string, error) {
+func (ls *LDKService) RedeemOnchainFunds(ctx context.Context, toAddress string, amount uint64, feeRate uint64, sendAll bool) (string, error) {
+	var fee *ldk_node.FeeRate
+	fee = ldk_node.FeeRateFromSatPerVbUnchecked(feeRate)
```

Comment:
@im-adithya  I think we should only do this if a `feeRate` is set. By default we should not set the fee rate and we should let the node decide.

Reply:
Done (changed to be a pointer)